app/assets/stylesheets/active_admin/components/_status_tags.scss

```diff
@@ -1,16 +1,15 @@
 .status_tag {
-  background: darken($secondary-color, 15%);
-  color: #fff;
-  text-transform: uppercase;
```

Comment:
No more nitpicking on styles I promise. You've done an amazing job, this is my only gripe :-)